### ISSUE TITLE
a way to check if a menu is open?

### ISSUE THREAD

**Issue by Pesoen** [ThymonA/menuv#40]
i made a simple vehicle control menu using MenuV, and it looks so good and uses so little resources compared to my other two menu's that did the same. my only issue is people want to use the same key to get the menu to show, to close it again. so i need a way to get a bool if the menu is showing and then do stuff accordingly. so the same button can close the menu as is showing it(using an if statement not the MenuV way, as i need it to only show in vehicles)

tl;dr a way to return a bool if a menu with a certain name is showing

**Comment by Mobius1** [comment]
```lua
    local menu = MenuV:CreateMenu('MenuV', 'Welcome to MenuV', 'topleft', 255, 0, 0, 'size-125', 'example', 'menuv', 'example_namespace')

    if menu.IsOpen then
        --
    end
```

**Comment by Pesoen** [comment]
awesome! thanks for the help :)
